hack/jenkins/common.sh

```diff
@@ -91,7 +91,7 @@ else
 fi
 
 # installing golang so we could do go get for gopogh
-sudo ARCH="$ARCH"./installers/check_install_golang.sh "1.16.4" "/usr/local" || true
+ARCH="$ARCH" ./installers/check_install_golang.sh "1.16.4" "/usr/local" || true
```

Comment:
ARCH="$ARCH" is no longer needed I'm pretty sure since, it's not under a sudo anymore.

Reply:
yep, good catch. fixed